tools/azure-pipelines/uploading_watchdog.sh

```diff
@@ -27,6 +27,9 @@ if [ -z "$HERE" ] ; then
   exit 1
 fi
 
+sudo sed -i 's/azure.archive.ubuntu.com/archive.ubuntu.com/g' /etc/apt/sources.list
+sudo apt-get update
+
```

Comment:
I‘m not sure this works as the azure service has recovered  when this CI running

Reply:
It did: [Here](https://dev.azure.com/apache-flink/apache-flink/_build/results?buildId=45803&view=logs&j=bea52777-eaf8-5663-8482-18fbc3630e81&t=b2642e3a-5b86-574d-4c8a-f7e2842bfb14&l=82) you can see that we're using a different mirror. This was just one draft PR to check whether that would be a feasible workaround. But I'm gonna close this one in favor of PR #21873 which uses mirrors.